src/local/butler/constants.py

```diff
@@ -50,6 +50,8 @@
   ABIS = {'linux': 'cp39', 'windows': 'cp39', 'macos': 'cp39'}
 elif sys.version_info.major == 3 and sys.version_info.minor == 10:
   ABIS = {'linux': 'cp310', 'windows': 'cp310', 'macos': 'cp310'}
+elif sys.version_info.major == 3 and sys.version_info.minor == 11:
+  ABIS = {'linux': 'cp310', 'windows': 'cp310', 'macos': 'cp310'}
```

Comment:
These don't need to be cp311?

Reply:
310 works fine on my computer.
Changing to 311 now (which also works).

Reply:
Is it possible, this "worked" because the ABIs are only used during deploy (so really cp310 would fail if actually used with 3.11)

Reply:
Yes, it could be. I did not deploy yet.